generators/voxel_generator_noise.cpp

```diff
@@ -1,16 +1,21 @@
 #include "voxel_generator_noise.h"
 
+void VoxelGeneratorNoise::set_noise(Ref<OpenSimplexNoise> noise) {
+	_noise = noise;
```

Comment:
Technically it should emit `changed` for any property modification, although right now nothing uses that

Reply:
Yes. And it's only being used here to alert VT so it can set the editor warning flag.